src/backend/services/metrics.py

```diff
@@ -62,25 +65,14 @@ class MetricsMiddleware(BaseHTTPMiddleware):
     """
 
     async def dispatch(self, request: Request, call_next: Callable):
-
-        self._confirm_env()
```

Comment:
why did we remove this? It'll help identify metrics config errors easily

Reply:
My bad, I removed to test locally